devicetypes/smartthings/smartsense-button.src/smartsense-button.groovy

```diff
@@ -262,18 +267,21 @@ def refresh() {
     } else {
         refreshCmds += zigbee.readAttribute(zigbee.POWER_CONFIGURATION_CLUSTER, 0x0020)
     }
-    refreshCmds += zigbee.readAttribute(zigbee.TEMPERATURE_MEASUREMENT_CLUSTER, 0x0000) +
-        zigbee.readAttribute(zigbee.IAS_ZONE_CLUSTER, zigbee.ATTRIBUTE_IAS_ZONE_STATUS) +
-        zigbee.enrollResponse()
 
+    if(device.getDataValue("manufacturer") != "HEIMAN") {
+        refreshCmds +=  zigbee.readAttribute(zigbee.TEMPERATURE_MEASUREMENT_CLUSTER, 0x0000)
```

Comment:
I missed that this device doesn't support temperature measurement. That unfortunately means we cannot use this DTH either. 

@varzac can you think of any other options besides the new DTH approach?

Reply:
At this point in time no.  Anything else would be hacky and would require local changes anyways.

Reply:
@FenMei007S I'm sorry this didn't work out but it seems like we have no choice but to create a new DTH for this device. Please try to match the Zigbee Button implementation as close as you can. Perhaps call it `Zigbee Non-Holdable Button`. I don't like that name so open to other ideas.